pr_agent/algo/utils.py

```diff
@@ -474,4 +474,13 @@ def clip_tokens(text: str, max_tokens: int, add_three_dots=True) -> str:
         return clipped_text
     except Exception as e:
         get_logger().warning(f"Failed to clip tokens: {e}")
-        return text
\ No newline at end of file
+        return text
+
+def replace_code_tags(text):
+    """
+    Replace odd instances of ` with <code> and even instances of ` with </code>
+    """
+    parts = text.split('`')
```

Comment:
i would add a check first to see if '`' is in the original text, and if not, return the input